content/en/user-manual/publishing/web/index.md

```diff
@@ -8,12 +8,10 @@ PlayCanvas games are actually web pages and as such, are most at home in the bro
 
 * [PlayCanvas Hosting][1]
 * [Self-hosting][2]
-* [Chrome Web Store][3]
-* [Kongregate][4]
-* [Facebook][5]
+* [Kongregate][3]
```

Comment:
I don't think Kongregate are accepting new games from developers anymore

Reply:
I've removed the Kongregate page.